app/models/service_template_orchestration.rb

```diff
@@ -3,6 +3,24 @@ class ServiceTemplateOrchestration < ServiceTemplate
 
   before_save :remove_invalid_resource
 
+  def self.create_catalog_item(options, _auth_user = nil)
+    transaction do
+      create(options.except(:config_info)).tap do |service_template|
+        config_info = options[:config_info]
+
+        service_template.orchestration_template = if config_info[:template_id]
```

Comment:
Do we want to validate both template and manager must exist? If this is the only call we should unless users can call update method to set them later. This make me think of the need for update method and delete method as a complete solution.